### ISSUE TITLE
write_dta fails with IndexError if only missing_user_values in series

### ISSUE THREAD

**Issue by eirki** [Roche/pyreadstat#159]
**Describe the issue**
When saving dataframe to DTA, if a series contains only a letter which is included in `missing_user_values`, the operation fails.

**To Reproduce**
```python
import pandas as pd
import pyreadstat

df = pd.DataFrame({"var": ["a", "a", "a", "a"]})
pyreadstat.write_dta(
    df=df,
    dst_path="deleteme.dta",
    missing_user_values={"var": ["a"]},
)
```

```
Traceback (most recent call last):
    pyreadstat.write_dta(
  File "pyreadstat/pyreadstat.pyx", line 835, in pyreadstat.pyreadstat.write_dta
  File "pyreadstat/_readstat_writer.pyx", line 584, in pyreadstat._readstat_writer.run_write
  File "pyreadstat/_readstat_writer.pyx", line 244, in pyreadstat._readstat_writer.get_pandas_column_types
  File "/home/ebs/venvs/survey-stat-engine/lib/python3.9/site-packages/pandas/core/indexing.py", line 895, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
  File "/home/ebs/venvs/survey-stat-engine/lib/python3.9/site-packages/pandas/core/indexing.py", line 1501, in _getitem_axis
    self._validate_integer(key, axis)
  File "/home/ebs/venvs/survey-stat-engine/lib/python3.9/site-packages/pandas/core/indexing.py", line 1444, in _validate_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds
```

**Expected behavior**
The file should be saved, with the specified variable as a numeric variable full of user missing values.

**Setup Information:**
System
pandas==1.2.4
pyreadstat==1.1.4
OS: Ubuntu 20.04

**Comment by ofajardo** [comment]
solved on dev

**Comment by ofajardo** [comment]
solved in version 1.1.5
